packages/gcf-utils/src/gcf-utils.ts

```diff
@@ -438,4 +448,53 @@ export class GCFBootstrapper {
       });
     }
   }
+
+  /*
+   * Setting the process.env.WEBHOOK_TMP environemtn variable indicates
+   * that the webhook payload should be written to a tmp file in Cloud
+   * Storage. This allows us to circumvent the 100kb limit on Cloud Tasks.
+   *
+   * @param body
+   */
+  private async maybeWriteBodyToTmp(body: string): Promise<string> {
+    if (process.env.WEBHOOK_TMP) {
+      const tmp = `${Date.now()}-${v4()}.txt`;
+      const bucket = storage.bucket(process.env.WEBHOOK_TMP);
+      const writeable = bucket.file(tmp).createWriteStream();
+      logger.info(`uploading payload to ${tmp}`);
+      writeable.write(body);
+      writeable.end();
+      await new Promise((resolve, reject) => {
+        writeable.on('error', reject);
+        writeable.on('finish', resolve);
+      });
+      return JSON.stringify({
+        tmpUrl: tmp,
+      });
+    } else {
+      return body;
+    }
+  }
+
+  /*
+   * If body has the key tmpUrl, download the original body from a temporary
+   * folder in Cloud Storage.
+   *
+   * @param body
+   */
+  private async maybeDownloadOriginalBody(payload: {
+    [key: string]: string;
+  }): Promise<object> {
+    if (payload.tmpUrl) {
+      if (!process.env.WEBHOOK_TMP) throw Error('no tmp directory configured');
```

Comment:
I need to find a prettier rule for this :) Please use curly braces and new lines here.